zomes/deepkey/Cargo.toml

```diff
@@ -9,8 +9,7 @@ name = "deepkey"
 
 [dependencies]
 hc_deepkey_types = { workspace = true }
-hdi = { workspace = true }
-hdk = { workspace = true }
+hdi = { workspace = true, features = ["trace"] }
```

Comment:
Should not depend on the HDK in integrity zomes. Tracing is provided by the HDI, it's just behind a feature. That's all that the HDK was being used for

Reply:
Thanks, that is great to know